signature/sign.go

```diff
@@ -107,6 +113,12 @@ func Sign(_ context.Context, key Key, sf SignedFielder, opts ...Option) (*pipeli
 		values[EnvNamespacePrefix+k] = v
 	}
 
+	for _, ignored := range options.ignoredEnvVars {
+		// Delete both the namespaced and non-namespaced versions of the var, just to be sure
+		delete(values, EnvNamespacePrefix+ignored)
+		delete(values, ignored)
```

Comment:
Is this right? This would delete `ignored` from the set of all data to sign, not limited to env vars. I'm thinking you actually wanted:
 
```suggestion
		delete(objEnv, ignored)
```